### PR TITLE
Add Merchants

### LINKED ISSUE #491: Request for merchants for JuditMatyus.com
Hi there!
www.juditmatyus.com now officially accepts Monero as a form of payment. Please add it to your list.
More info here 
www.juditmatyus.com/monero

Thanks!

### LINKED ISSUE #492: Create ATM section on https://getmonero.org/community/merchants
Hi, please create an ATM section on https://getmonero.org/community/merchants
Also you could put a link to https://trovemat.com/ that can sell Monero for Fiat Cash. And soon will be able to dispence Fiat Cash for Monero. 
Thank you!

### PR DESCRIPTION
**Pull request by QuickBASIC**
* Judit Mátyus ( closes #491 )
* Trovemat.com ( closes #492 )

### CODE REVIEW

File: _data/merchants.yml [hunk 1]
```diff
@@ -45,6 +45,8 @@
       url: https://shapeshift.io/
     - name: Tux Exchange
       url: https://www.tuxexchange.com/trade?coin=XMR&market=BTC
+      name: Trovemat.com (ATM)
```
Comment (MaxXor) [inline _data/merchants.yml]: A `-` is missing in front of the name. Also I'm not sure if Trovemat should be considered an exchange by itself as they are a merchant selling ATMs.

File: _data/merchants.yml [hunk 2]
```diff
@@ -45,6 +45,8 @@
...
+      url: https://trovemat.com/
```
Comment (MaxXor) [inline _data/merchants.yml]: Remove additional whitespace at the end.

File: _data/merchants.yml [hunk 3]
```diff
@@ -203,6 +205,8 @@
       url: https://synntechgaming.com
     - name: Xusah Boutique
       url: https://xusah.boutique/
+      name: Judit Mátyus - Jewelry
+      url: https://www.juditmatyus.com/
```
Comment (MaxXor) [inline _data/merchants.yml]: Remove additional whitespace at the end.

File: _data/merchants.yml [hunk 4]
```diff
@@ -203,6 +205,8 @@
       url: https://synntechgaming.com
     - name: Xusah Boutique
       url: https://xusah.boutique/
+      name: Judit Mátyus - Jewelry
```
Comment (MaxXor) [inline _data/merchants.yml]: A `-` is missing in front of the name.

**Comment by QuickBASIC** [comment]
Fixed.

**Comment by MaxXor** [comment]
Well, the many spaces after `https://www.juditmatyus.com/` are still there..

**Comment by QuickBASIC** [comment]
> Well, the many spaces after https://www.juditmatyus.com/ are still there..

Not sure how that happened. Fixed. I'll be more careful in the future to check the diff. Thanks.

**Comment by mattcode55** [comment]
@QuickBASIC Which editor do you use? Most have an option to automatically trim trailing whitespace when you save.

**Comment by MaxXor** [comment]
Would be really awesome if you squash all those little commits into one.

**Comment by luigi1111** [comment]
@QuickBASIC please squash.

**Comment by QuickBASIC** [comment]
@luigi1111 Done.

**Pull request merged**